sechub-wrapper-xray/src/test/java/com/mercedesbenz/sechub/wrapper/xray/api/XrayWrapperArtifactoryClientTest.java

```diff
@@ -34,22 +31,19 @@ class XrayWrapperArtifactoryClientTest {
     void beforeEach() {
         XrayWrapperArtifact artifact = new XrayWrapperArtifact("name", "sha256", "tag", XrayWrapperScanTypes.DOCKER);
         XrayWrapperConfiguration configuration = XrayWrapperConfiguration.Builder
-                .create("http://notmalformed-url-example.com", "example", "zipDirectory", "report").build();
+                .builder("http://notmalformed-url-example.com", "example", "zipDirectory", "report").build();
```

Comment:
mentioned before, but maybe on something else:

```suggestion
                .builder().
                     url("http://notmalformed-url-example.com").
                     type("example").
                     zipDirectory("zipDirectory").
                     reportDirectory("report").
                  build();
```

Remark: i just added some method names, do not just use them, the introduced names are just to show up the idea.

A builder should have dedicated methods for the parameters.
A builder is intended to avoid "hell of construction parameters" - so multiple parameters inside builder itself makes not much sense IMO.